docs/components/modeler/web-modeler/advanced-modeling/manage-connector-templates.md

```diff
@@ -0,0 +1,105 @@
+   - On the right, you see the live **Visual Preview**. The live preview shows how the properties panel will look when you apply the template to an element. It automatically updates on every valid change, and reflects the latest valid state of the template. The preview allows you to interactively check your template before publishing, including its usability.
+   - In the upper left, you can **Upload an icon** for your template. You can upload any image file you want, however we recommend to use squared SVG graphics. The icons get rendered 18x18 pixels in the element on the modeling canvas, and 32x32 pixels in the properties panel.
+
+   On every valid change, the template is saved automatically. In case of a problem, the template is not saved, and you need to [fix it beforehand](#fixing-template-problems).
+
+   ![Template editor anatomy](img/connector-templates/create-connector-template-2.png)
+
+4. Once you've written your template, you can **publish** it by clicking on the corresponding action. You will be prompted to optionally enter a version name. This name appears in your milestone list.
+
+   ![Publishing a template](img/connector-templates/create-connector-template-3.png)
+
+5. After publishing, the template can be applied to elements of all models in the same project.
+
+   :::info Project scope
+   A connector template is always associated with a project, hence only available in the project where you added it.
+   :::
+
+### JSON editor features
+
+The JSON editor is based on the [Monaco Editor](https://microsoft.github.io/monaco-editor/). The Monaco Editor is the editor that powers VS Code. As a result, the template editor supports many familiar features such as auto-formatting, indentation support, code completion, and error highlighting. Read the [VS Code editor docs](https://code.visualstudio.com/docs/editor/editingevolved) for a full overview of features.
+
+### Versioning connector templates
+
+You can version your connector templates using [milestones](/components/modeler/web-modeler/milestones.md), similar to diagrams.
+
+If you publish a new version of a connector template, and the old version was already used in diagrams, the user can decide to update to the most recent version, or to stay with the old version when editing the respective diagram element (see updating [element templates](https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/using-templates/#updating-templates)).
+
+## Fixing problems in your templates {#fixing-template-problems}
```

Comment:
Is there a particular reason why you've included the curly braces and text here?

Reply:
@christinaausley This is an anchor link and allows you to link to a specific section inside a single page